### ISSUE TITLE
History sync

### ISSUE THREAD

**Issue by Geraldziu** [tegon/traktflix#73]
Hi,

I just tried to sync my netflix history, and trakflix reported that some history is synced ie: "Movies 5 / Series 2". But when i logged into trakt i found out that nothing new was synced.

Trakflix 1.0.8.3
Netflix language: Polish

**Comment by gsrafael01** [comment]
This is a bit old, but I think it was related to the extension only working for English titles in the past. Could you try in v2.0.8 and see if it's still an issue?

**Comment by Geraldziu** [comment]
look's like it's working now :) a little bit slow if I want to load my whole history. It would be a good idea to add some way of moving to specific page or maybe a filter like "watched between dd-mm-yyy and dd-mm-yyyy"

**Comment by gsrafael01** [comment]
Is it actually loading though? Some users reported that if they try to load more than 10 pages at once it's currently freezing the browser.

**Comment by Geraldziu** [comment]
Yes it's working - or at least it worked one time to load about 70 pages. But i couldn't make it work second time. Maybe if I'd wait for longer time. 
I think that the problem is with loading trakt data, netflix history is loaded after about 1 min, but then loading and matching with trakt is very slowly and without any indication that it's working and not frozen.

**Comment by mykutu** [comment]
i can see https://www.netflix.com/WiViewingActivity
But when i click Trackflix / History get in stuck " Loading page 1... "

**Comment by alexanderadam** [comment]
> Is it actually loading though? Some users reported that if they try to load more than 10 pages at once it's currently freezing the browser.

How can I trigger the history sync?
When I go to the history page of the addon, I'm seeing `There's no more items in your history. Refresh the page to go to the beginning.`.
When I open developer tools, I'm seeing that the request to Netflix resulted in `401` Unauthorized. But with right click `Open in New Tab` I can see and access the JSON response. So I guess this might be related to a header and/or authorization issue.

I'm using Firefox 82, which is also affected by bug #132.

**Comment by goestav** [comment]
Something is wrong with the values of the **NetflixId** and **SecureNetflixId** cookies.
Just changed them to the cookie values I get from opening in a new tab and I got a 200 response code.
<img width="630" alt="image" src="https://user-images.githubusercontent.com/27970303/98138920-83917e00-1ebb-11eb-9d04-dc43016b347b.png">
I use Firefox 83 (on macOS)

**Comment by rafaelgomesxyz** [comment]
@alexanderadam @goestav Since you guys are on Firefox, do you have any privacy settings enabled that could prevent the extension from being able to send your cookies with the request? In particular, when using Firefox containers, you need to grant the extension permission to handle your cookies for it to work properly (you can do this in the options page).

**Comment by goestav** [comment]
Just found out the Multi-Account Containers extension was causing the problem :/
Enabling the cookie permissions option didn't change anything for me, but opening the extension page in the same container as Netflix works.

**Comment by alexanderadam** [comment]
@goestav, nice finding. So it's issue #132 after all, because I guess the extension would use the same container if the button would be working?

**Comment by goestav** [comment]
@alexanderadam, is the traktflix logo not showing up for you? It only pops up when you're on [Netflix](https://www.netflix.com).
It should show in the address bar
![image](https://user-images.githubusercontent.com/27970303/98244843-f6f0c980-1f67-11eb-9795-e0e14d735ab3.png)

**Comment by alexanderadam** [comment]
Ah indeed, it does!
Opening the options screen does not work though and the history won't be opened in the same container.

**Comment by goestav** [comment]
Open the extension page (`moz-extension://` url) in the same container by default if you always use the same  one.
The options button goes to a blank page for some reason but it works when holding `Cmd ⌘` or `Ctrl ⌃`.
